Lib/typing.py

```diff
@@ -3416,6 +3417,9 @@ class CustomerModel(ModelBase):
         assumed to be True or False if it is omitted by the caller.
     - ``kw_only_default`` indicates whether the ``kw_only`` parameter is
         assumed to be True or False if it is omitted by the caller.
+    - ``frozen_default`` indicates whether the ``frozen`` parameter is
+        assumed to be True or False if it is omitted by the caller. Only
+        supported in Python 3.12+.
```

Comment:
```suggestion
        assumed to be True or False if it is omitted by the caller.
```
I don't think this is meaningfully true, since type checkers will support it in any version, and becaus we added `**kwargs`, it will work at runtime in 3.11 too.